spec/graphql/wiki_page_edit_history_spec.rb

```diff
@@ -25,9 +25,9 @@
     wiki_page.save
   end
 
-  it "shows the initial wiki page creation as both changed" do
-    expect(edits.last["changeSet"]["bodyChanged"]).to eql true
-    expect(edits.last["changeSet"]["frontMatterChanged"]).to eql true
+  it "shows the initial wiki page creation as neither changed" do
```

Comment:
this is a change in behavior as a result of fixing this bug. we'll need to update the logic in the history page to accommodate that